packages/runtime/state/components-meta.js

```diff
@@ -0,0 +1 @@
+module.exports = require("../dist/state/components-meta.cjs")
```

Comment:
No newline at the end of the file (although I'm guessing this file will be deleted)